src/content/docs/en/reference/api-reference.mdx

```diff
@@ -1490,18 +1696,25 @@ export function onRequest (context, next) {
 
 ### `next()`
 
-A function that intercepts (reads and modifies) the `Response` of a `Request` or calls the "next" middleware in the chain and returns a `Response`. For example, this function could modify the HTML body of a response.
+<p>
 
-This is an optional argument of `onRequest()`, and may provide the required `Response` returned by the middleware.
+**Type:** `(rewritePayload?: string | URL | Request) => Promise<Response>`<br />
+<Since v="4.13.0" />
+</p>
 
-<p><Since v="4.13.0" /></p>
+A function that intercepts (reads and modifies) the `Response` of a `Request` or calls the "next" middleware in the chain and returns a `Response`. For example, this function could modify the HTML body of a response.
 
-**Type:** `(rewritePayload?: string | URL | Request) => Promise<Response>`
+This is an optional argument of `onRequest()`, and may provide the required `Response` returned by the middleware.
 
 It accepts an optional URL path parameter in the form of a string, `URL`, or `Request` to [rewrite](/en/guides/routing/#rewrites) the current request without retriggering a new rendering phase.
```

Comment:
i.e. something like this:

```suggestion
Since Astro v4.13.0, `next()` accepts an optional URL path parameter in the form of a string, `URL`, or `Request` to [rewrite](/en/guides/routing/#rewrites) the current request without retriggering a new rendering phase.
```

Reply:
I didn't read your suggestion before answering to the other comment. This suggestion seems better if we want to keep consistency with the other sections. The [current `Added in:`/`Type:` in the middle of the section](https://docs.astro.build/en/reference/api-reference/#next) seems a little odd.

So should I revert my changes or we use your suggestion?

Reply:
I think I like my suggestion 😄 

I agree with you that it’s weird to have the “Added in” in the middle of the paragraph — and even weirder to have the type there. It makes it kind of unclear what is being referred to.

Alternatively, the current style could maybe make more sense with a subheading?

<img width="756" alt="next() API reference section with separate heading for next(requestPayload)" src="https://github.com/user-attachments/assets/3fbc00f3-81fb-4767-aabe-ac6557ba872c">

But I think this overcomplicates things. Keeping the type clearly up top is preferable in my opinion.

Reply:
Hum it could be confusing for users I think. They might think that the two are unrelated. I prefer your first suggestion and I think it's better to remove `Since` here. Usually `Since` applies to the section and not the parameter, so something like:

![Example of the restructured next() section](https://github.com/user-attachments/assets/8e665540-0e40-481f-8821-41dd1d565cc3)

Reply:
I just realized that this is exactly what you are suggesting when combined with the comment on `Since`. (I must be tired 😅)  So yes, I'm okay with those changes!